File: backend/src/main/java/com/mulesoft/arc/arcdatastore/backend/models/ArcAnalyticsMonthlyUsers.java
package com.mulesoft.arc.arcdatastore.backend.models;

/**
 * A class representing Daily Users type.
 */

public class ArcAnalyticsMonthlyUsers extends ArcAnalyticsRangeResult {

    public ArcAnalyticsMonthlyUsers(String day) {
        super("ArcAnalytics#MonthlyUsers", day);
    }
}

File: models/arc-analytics-monthly-users.js
'use strict';

const {ArcAnalyticsRangeResult} = require('./arc-analytics-range-result');

/**
 * A class representing monthly Users type.
 */
class ArcAnalyticsMonthlyUsers extends ArcAnalyticsRangeResult {
  /**
   * @param {String} startDay The start day of the date range.
   * @param {String} endDay The last day of the date range.
   * @param {Number} result The result of computation.
   */
  constructor(startDay, endDay, result) {
    super('ArcAnalytics#MonthlyUsers', startDay, endDay, result);
  }
}

module.exports.ArcAnalyticsMonthlyUsers = ArcAnalyticsMonthlyUsers;
